dev/app.js

```diff
@@ -497,6 +499,16 @@ renderSiteApp(
                     pathname: "/bubble/graph-and-legend-padding-reduced",
                     content: renderBubbleGraphAndLegendPaddingReduced,
                     title: "Graph and Legend Padding Reduced"
+                },
+                {
+                    pathname: "/bubble/single-dataset-api",
+                    content: renderBubbleSingleDataset,
+                    title: "Single Dataset API"
+                },
+                {
+                    pathname: "/bubble/multiple-dataset-api",
+                    content: renderBubbleMultipleDataset,
```

Comment:
I think it would be better if we rename it to **renderBubbleGraphWithMultipleDataset**.

Reply:
fixed here:
https://github.com/cerner/carbon-graphs/pull/308/commits/100e84b9e54c438b8e2bf616e2ac7ea4fff6080b